### PR TITLE
Rename `@ignore` directive to `@computed`, and `dependsOn` argument to `from`

### PR DESCRIPTION
**Pull request by darrellwarde**
# Description

Renames `@ignore` to `@computed`. As far as we're aware, there are no other use cases of `@ignore` than to resolve them with a custom resolver. As such, this rename makes sense.

# Issue

#937

### CODE REVIEW

**Comment by neo4j-team-graphql** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-graphql-docs-979.surge.sh

File: packages/graphql/src/types.ts [hunk 1]
```diff
@@ -123,7 +123,7 @@ export interface BaseField {
     description?: string;
     readonly?: boolean;
     writeonly?: boolean;
-    ignored?: boolean;
+    // ignored?: boolean;
```
Comment (angrykoala) [inline packages/graphql/src/types.ts:126]: :eyes: 
```suggestion
```

**Review by angrykoala — APPROVED** [review]

File: packages/graphql/src/types.ts (hunk 1 above)
Comment (darrellwarde) [reply packages/graphql/src/types.ts]: 😳

**Comment by nelsonpecora** [comment]
Oh, I think @dmoree mentioned that `@computed` might have unintended connotations. Do you still feel strongly about that?

**Comment by nelsonpecora** [comment]
@dmoree `@custom` makes sense to me! 👍🏼👍🏼

**Pull request merged**